README.md

```diff
@@ -6,31 +6,47 @@ Hound is an extremely fast source code search engine. The core is based on this
 [Regular Expression Matching with a Trigram Index](http://swtch.com/~rsc/regexp/regexp4.html). Hound itself is a static
 [React](http://facebook.github.io/react/) frontend that talks to a [Go](http://golang.org/) backend. The backend keeps an up-to-date index for each repository and answers searches through a minimal API. Here it is in action:
 
-![Hound Screen Capture](screen_capture.gif)
+![Hound Screen Capture](imgs/screen_capture.gif)
 
 ## Quick Start Guide
 
 ### Using Go Tools
 
+0. [Install the Go tools](https://golang.org/doc/install) if you don't have it. 
+
++ After installing Go, make sure go's version is 1.4+ by typing `go version` in the command line. 
++ Set up your [`GOPATH`](https://github.com/golang/go/wiki/GOPATH) to a directory where you want to store all your Go packages. Otherwise
+all packages will be installed to `$HOME/go` by default.
+
 1. Use the Go tools to install Hound. The binaries `houndd` (server) and `hound` (cli) will be installed in your $GOPATH.
 
 ```
 go get github.com/hound-search/hound/cmds/...
 ```
 
-2. Create a [config.json](config-example.json) in a directory with your list of repositories.
+2. Create a [simple-config-example.json](config-examples/simple-config-example.json) in your local file directory. For simplicity, we just showed 
```

Comment:
I think that the quick start guide for the most part should be kept relatively brief. The `config-example.json` documents the variety of ways to set up a repo in lieu of more explicit documentation. Since we don't really know what sort of repo people want to track, it's probably a good idea to not make a ton of assumptions. 

Instead, it might be helpful to use the `default-config.json` that's included in the root of the repo:
> Create a `config.json` file and use it to list your repositories. Check out [our example `config.json`](#link) to see how to set up various types of repositories. For example, we can configure Hound to search its own source code using the config found in `default-config.json`:
```json
{
  "dbpath" : "db",
  "repos" : {
    "Hound" : { "url" : "https://github.com/etsy/hound.git" }
  }
}
```

Reply:
I see. thank you! I did not notice there was a simplified `default-config.json` before.